beta-src/src/webDiplomacyTheme.ts

```diff
@@ -113,6 +128,34 @@ const webDiplomacyTheme = createTheme({
       main: "#fff",
       contrastText: "#000",
     },
+    France: {
+      main: "#2D5EE8",
+      light: "#B9C9F7",
+    },
+    Austria: {
+      main: "#FC4343",
+      light: "#FEC0C0",
+    },
+    England: {
+      main: "#E136EA",
+      light: "#F5BCF8",
+    },
+    Germany: {
+      main: "#F37C0E",
+      light: "#F5BCF8",
+    },
+    Russia: {
+      main: "#3F1BC1",
+      light: "#BFB3EA",
+    },
+    Italy: {
+      main: "#47D2A0",
+      light: "#C2F0DF",
+    },
+    Turkey: {
+      main: "#F3C400",
+      light: "#FBEBAA",
+    },
```

Comment:
use the country enum and create a type for this and then add it to the theme palette after the declaration.